websites/A/AnimeHeaven/presence.ts

```diff
@@ -0,0 +1,143 @@
+const presence = new Presence({
+    clientId: "816042675626442783"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+type VideoData = {
+  paused: boolean;
+  currentTime: number;
+  duration: number;
+};
+let iFrameVideo: VideoData;
+presence.on("iFrameData", (videoData: VideoData) => {
+  iFrameVideo = videoData;
+});
+
+const paths = {
+  async "/watch"(presenceData: PresenceData, buttons: boolean) {
+    const title = document.querySelector<HTMLAnchorElement>(".now2 .c a")
+        ?.textContent,
```

Comment:
It had to be ``HTMLElement`` not ``HTMLAnchorElement``

Reply:
I'm getting an anchor though?

Reply:
Oh right 🤦 my bad